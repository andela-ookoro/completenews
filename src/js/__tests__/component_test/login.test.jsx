import React from 'react';
import { mount, shallow } from 'enzyme';
import Renderer from 'react-test-renderer';
import Login from '../../pages/login';

test('Component render the article template', () => {
  const component = Renderer.create(<Login />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Should render a link to login when user is anonymous',
  () => {
    const wrapper = shallow(<Login />);
    // const loginText = wrapper.find('a').text();
    // expect(loginText).toEqual('Click to login');
    console.log(wrapper);
  });

