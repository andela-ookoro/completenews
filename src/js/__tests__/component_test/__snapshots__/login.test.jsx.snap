// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component render the article template 1`] = `
<a
  className=""
  onClick={[Function]}
  style={Object {}}
>
  Click to login
</a>
`;
